llvm/tools/llc/llc.cpp

```diff
@@ -365,15 +365,10 @@ int main(int argc, char **argv) {
   }
 
   // RemoveDIs debug-info transition: tests may request that we /try/ to use the
-  // new debug-info format, if it's built in.
-#ifdef EXPERIMENTAL_DEBUGINFO_ITERATORS
-  if (TryUseNewDbgInfoFormat) {
-    // If LLVM was built with support for this, turn the new debug-info format
-    // on.
+  // new debug-info format.
+  if (TryUseNewDbgInfoFormat)
+    // Turn the new debug-info format on.
```

Comment:
Nit, comments are counted when deciding whether an `if` needs braces; same applies everywhere else this is repeated.

Reply:
Fixed